libs/ui-lib/lib/cim/components/Hypershift/DetailsPage/HypershiftKubeconfigDownload.tsx

```diff
@@ -16,7 +16,7 @@ const HypershiftKubeconfigDownload = ({
 }: HypershiftKubeconfigDownloadProps) => {
   const { t } = useTranslation();
   const kubeconfigSecretName = hostedCluster.status?.customkubeconfig
-    ? hostedCluster.status.customkubeconfig?.name
+    ? hostedCluster.status.customKubeconfig?.name
```

Comment:
You'll need to update all occurrences, like the line above it, and the type definition.